materialize-mongodb/Dockerfile

```diff
@@ -28,7 +28,7 @@ ENV PATH="/connector:$PATH"
 
 # Bring in the compiled connector artifact from the builder.
 COPY --from=builder /builder/connector /connector/materialize-mongodb
-COPY --from=builder /lib/x86_64-linux-gnu/libgcc_s.so.1 /lib/x86_64-linux-gnu/
```

Comment:
I assume this old shared-library copy was no longer in use?

Reply:
Yeah, seems like a copy/paste from elsewhere perhaps, but I couldn't find any reason for it so decided to clean it up.